editor/test/editor/console_test.clj

```diff
@@ -134,8 +134,11 @@
                        ["   via '/main.lua:8', '/main.lua:13'"]))))
 
 (deftest three-matches-on-same-line-where-two-are-partial
-  (is (= 3 (count
-             (#'console/make-line-sub-regions {"/absolute/project/path/file.lua" :WIN
-                                               "/some/really/long/path/to/some/resource/in/the/project/the_resource.script" :WIN}
-                                              identity 10
-                                              "urce/in/the/project/the_resource.script:25: in function <urce/in/the/project/the_resource.script:24> </absolute/project/path/file.lua:65>")))))
+  (let [resource-map {"/absolute/project/path/file.lua" :WIN
+                      "/some/really/long/path/to/some/resource/in/the/project/the_resource.script" :WIN}]
+   (is (= 3 (count
+              (#'console/make-line-sub-regions
+                resource-map
+                (console/make-resource-suffix-map-delay resource-map)
+                identity 10
+                "...to/some/resource/in/the/project/the_resource.script:25: in function <...to/some/resource/in/the/project/the_resource.script:24> </absolute/project/path/file.lua:65>"))))))
```

Comment:
The syntax changed, so the test was updated to reflect that. The change is: we need at least 50 characters (the size of the suffix key) after `...` for the suffix search to work. Currently, the ellipsis-prefixed paths are 56 characters long, but I wanted to have some wiggle room in case there were some minor differences in behavior between e.g. different platforms or architectures. The shortened paths in this test were less than 50 characters.